server/fleet/queries.go

```diff
@@ -118,8 +118,8 @@ type QuerySpec struct {
 	Query       string `json:"query"`
 }
 
-func LoadQueriesFromYaml(yml string) ([]*Query, error) {
-	queries := []*Query{}
+func LoadQueryObjectsFromYaml(yml string) ([]*QueryObject, error) {
+	var queryObjects []*QueryObject
```

Comment:
I think changes in this file are no longer necessary?

Reply:
thanks @zwass , forgot about those changes.